commands/create_cluster_test.go

```diff
@@ -7,6 +7,8 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/giantswarm/gsctl/commands/errors"
+	"github.com/giantswarm/gsctl/flags"
```

Comment:
Internal import to be ordered below 3rd party

Reply:
Changed